portfolio/src/main/webapp/index.html

```diff
@@ -220,6 +220,26 @@ <h3>Upload an image:</h3>
                     <button>Submit</button>
                 </form>
             </div>
+            <div id="translation-div">
+                <h2>Translation</h2>
+                <p>Select a language, then click translate to get a translation of all the comments.</p>
+
+                <select id="language" required>
+                <option value="en">English</option>
+                <option value="zh">Chinese</option>
+                <option value="es">Spanish</option>
+                <option value="hi">Hindi</option>
+                <option value="ar">Arabic</option>
+                <option value="fr">French</option>
+                <option value="de">German</option>
+                <option value="ko">Korean</option>
+                </select>
+
+                <button onclick="requestTranslation();">Translate</button>
+
+                <hr/>
+            </div>
```

Comment:
I would place this after the comments section as it belongs there.

Reply:
Ok will do that!